manon/components/form-fieldset.scss

```diff
@@ -16,79 +16,23 @@ But the alternative styling as you can see below.
 For more information on the bug see: https://github.com/philipwalton/flexbugs#9-some-html-elements-cant-be-flex-containers
 and: https://bugs.chromium.org/p/chromium/issues/detail?id=262679 */
 
-form,
-main section form,
-main article form,
-main div form {
-  > fieldset {
+form {
+  fieldset {
+    display: inline-block;
+    margin: 0;
     padding: theme.$form-fieldset-padding;
-
-    @include theming.apply-from-theme(
-      (
-        border-width: theme.$form-fieldset-border-width,
-        border-style: theme.$form-fieldset-border-style,
-        border-color: theme.$form-fieldset-border-color,
-      )
-    );
+    border-width: theme.$form-fieldset-border-width;
+    border-style: theme.$form-fieldset-border-style;
+    border-color: theme.$form-fieldset-border-color;
 
     legend {
-      @include theming.apply-from-theme(
-        (
-          font-weight: theme.$form-fieldset-legend-font-weight,
-          margin: theme.$form-fieldset-legend-margin,
-        )
-      );
+      padding: 0;
```

Comment:
We want the theme builders to be able to set any visual styling. We also want to be as least opinionated with base styling as we can be. Add it as a variable instead ideally without a base value. (default can be set to null)